index.html

```diff
@@ -47,6 +47,190 @@ <h2 class="headingTxt">I’m Sumeya. Glad to see you!</h2>
+           <h4>CANOPY</h4>
+           <ol class="projectOrderedList">
+            <li>Back End Dev</li>
+            <li>2015</li>
+           </ol>
+           </div>
+           <p class="projectDetail txtWrapper">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           <ul class="projectUnorderdList">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+           <div class="btnWrapper">
+           <button class="btn-project">See Project</button>
+          </div>
+          </div>
+         </div>
+
+         <div class="worksFlexContainer">
+          <div class="workImgContainer">
+           <img src="work3.png" alt="">
+          </div>
+          <div class="workDetailContainer">
+           <h1 class="projecTitle txtWrapper">Tonic</h1>
+           <div class="canopyContainer">
+           <h4>CANOPY</h4>
+           <ol class="projectOrderedList">
+            <li>Back End Dev</li>
+            <li>2015</li>
+           </ol>
+           </div>
+           <p class="projectDetail txtWrapper">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           <ul class="projectUnorderdList">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+           <div class="btnWrapper">
+           <button class="btn-project">See Project</button>
+          </div>
+          </div>
+         </div>
+
+         <div class="worksFlexContainer">
+          <div class="workImgContainer">
+           <img src="work4.png" alt="">
+          </div>
+          <div class="workDetailContainer">
+           <h1 class="projecTitle txtWrapper">Multi-Post
+            Stories</h1>
+           <div class="canopyContainer">
+           <h4>CANOPY</h4>
+           <ol class="projectOrderedList">
+            <li>Back End Dev</li>
+            <li>2015</li>
+           </ol>
+           </div>
+           <p class="projectDetail txtWrapper">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+           <ul class="projectUnorderdList">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+           <div class="btnWrapper">
+           <button class="btn-project">See Project</button>
+          </div>
+          </div>
+         </div>
+        </div>
+      </section>
+
+      <section id="about">
+        <div class="aboutMainContainer">
+          <div class="AboutContentContainer">
+            <h2 class="headingTxt">About Myself</h2>
+            <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="connectMeLink" href="">LET’S CONNECT</a>
```

Comment:
You are right, `Let's connect` looks like a link here but actually it is a heading element of the social links. Kindly use hierarchically correct hx elements for it to make the page more semantic.